### PR TITLE
Required mac support

### PR DESCRIPTION
**Pull request by mmoscher**

### CODE REVIEW

File: provisioning/tasks/install/Darwin/helm.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+- name: Install helm
+  include_tasks:
+    file: tasks/install/_install_versioned_binary.yml
+  vars:
+    name: helm
+    url: "https://get.helm.sh/helm-{{ tooling.version.helm }}-darwin-{{ machine_architecture }}.tar.gz"
+    version: "{{ tooling.version.helm }}"
+    unzip: true
+    extra_opts:
+      - "--strip-components=1"
```
Comment (PB1994) [inline provisioning/tasks/install/Darwin/helm.yml:11]: newline

File: provisioning/tasks/install/Debian/helm.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- name: Install helm
+  include_tasks:
+    file: tasks/install/_install_versioned_binary.yml
+  vars:
+    name: helm
+    url: "https://get.helm.sh/helm-{{ tooling.version.helm }}-linux-amd64.tar.gz"
+    version: "{{ tooling.version.helm }}"
+    unzip: true
+    extra_opts:
+      - "--strip-components=1"
```
Comment (PB1994) [inline provisioning/tasks/install/Debian/helm.yml:11]: newline

File: provisioning/tasks/install/_install_module_if_existing.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    path: "tasks/install/general/{{ module }}.yml"
+  register: general_module_result
+
+- name: Include os specific module tasks for {{ module }} iff existing
+  include_tasks: "tasks/install/{{ ansible_facts['os_family'] }}/{{ module }}.yml"
+  when: module_result.stat.exists
+
+- name: Include general module tasks for {{ module }} iff os specific are missing
+  include_tasks: "tasks/install/general/{{ module }}.yml"
+  when: not module_result.stat.exists and general_module_result.stat.exists
```
Comment (PB1994) [inline provisioning/tasks/install/_install_module_if_existing.yaml:17]: newline

File: provisioning/vars/main.yml [hunk 4]
```diff
@@ -63,22 +70,22 @@ dev:
     - name: 1password
       is_classic: false
       channel: edge
-    - name: kubectl
-      is_classic: true
-    - name: helm
-      is_classic: true
   remove_packages:
     - helm
```
Comment (PB1994) [inline provisioning/vars/main.yml:74]: ?

**Review by PB1994 — CHANGES_REQUESTED** [review]

File: provisioning/vars/main.yml (hunk 4 above)
Comment (mmoscher) [reply provisioning/vars/main.yml:74]: passt

File: provisioning/tasks/install/Darwin/helm.yml (hunk 1 above)
Comment (mmoscher) [inline provisioning/tasks/install/Darwin/helm.yml:11]:
```suggestion
      - "--strip-components=1"

```

File: provisioning/tasks/install/Debian/helm.yml (hunk 2 above)
Comment (mmoscher) [inline provisioning/tasks/install/Debian/helm.yml:11]:
```suggestion
      - "--strip-components=1"

```

File: provisioning/tasks/install/_install_module_if_existing.yaml (hunk 3 above)
Comment (mmoscher) [inline provisioning/tasks/install/_install_module_if_existing.yaml:17]:
```suggestion
  when: not module_result.stat.exists and general_module_result.stat.exists

```

**Pull request merged**